mimic/rest/cinder_api.py

```diff
@@ -31,7 +31,14 @@ def catalog_entries(self, tenant_id):
         """
         return [
             Entry(
-                tenant_id, "volume", "cloudBlockStorage",
```

Comment:
True...

From working on Cloud Backup and now working through a migration of API versions, I know there's a field in the Service Catalog entry that provides for versioning capabilities - in fact, I use it with the Endpoint Templating functionality I put into Mimic. I don't know that `IAPIMock` supports it yet though.